app/views/dashboard/advisers/_advisers_table.html.erb

```diff
@@ -1,28 +1,36 @@
-<div class="table-wrapper">
-    <table class="l-half-width">
-      <thead>
+<div class="table-wrapper js-filter-wrapper">
+  <div class="l-top-right-action">
+    <%= add_adviser_button(firm: firm) %>
+  </div>
+
+	<% if title.present? %>
+		<h2><%= title %></h2>
+	<% end %>
```

Comment:
I'd prefer we keep `_advisers_table.html.erb` just as the table. Apart from anything else — we'd expect `.table-wrapper` to wrap a table, not headers and buttons and such. 

I can see why you've done it — because we need to hide the headers too — but I think we have to solve that via another refactor. If we _do_ need to hide those headings, we need to rename `FilterTable`, since it's no longer just filtering tables but doing other stuff too.